mleap-core/src/test/scala/ml/combust/mleap/core/feature/MaxAbsScalerModelSpec.scala

```diff
@@ -12,10 +14,9 @@ class MaxAbsScalerModelSpec extends FunSpec {
     val scaler = MaxAbsScalerModel(Vectors.dense(Array(20.0, 10.0, 10.0, 20.0)))
 
     it("Scales the vector based on absolute max value"){
-      val inputArray = Array(15.0, -5.0, 5.0, 19.0)
-      val expectedVector = Array(0.75, -0.5, 0.5, 0.95)
-
-      assert(scaler(Vectors.dense(inputArray)).toArray.sameElements(expectedVector))
+      val inputVector = Vectors.dense(15.0, -5.0, 5.0, 19.0)
+      val expectedVector = Vectors.dense(0.75, -0.5, 0.5, 0.95)
+      assert(scaler(inputVector) ~= expectedVector relTol 1E-3)
```

Comment:
See my comment below, 1E-3 seems too big, is there an actual reason why this cannot be something like 1E-9 or 1E-12?